recipes/glib/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set major_minor = "2.48" %}
+{% set version = major_minor + ".0" %}
+{% set sha256 = "744be6931ca914c68af98dc38ff6b0cf8381d65e335060faddfbf04c17147c34" %}
+
+package:
+  name: glib
+  version: {{ version }}
+
+source:
+  fn: glib-{{ version }}.tar.xz
+  url: http://ftp.acc.umu.se/pub/GNOME/sources/glib/{{ major_minor }}/glib-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+  # patches forked from https://github.com/pkgw/conda-recipes
+  #  Copyright 2014-2015 Peter Williams and collaborators.
+  # This file is licensed under a 3-clause BSD license.
+  patches:
+    - system-data-dirs.patch
+    - no-cocoa-notification.patch  # [osx]
+    - find-libintl.h.patch  # [osx]
```

Comment:
These patches are probably ok, but we should try and take a look at Homebrew's [formula]( https://github.com/Homebrew/legacy-homebrew/blob/master/Library/Formula/glib.rb ) too. I think some of these overlap.

* Hardcoded paths patch ( like `system-data-dirs.patch` ) -- https://raw.githubusercontent.com/Homebrew/patches/59e4d32/glib/hardcoded-paths.diff
* No notification patch ( like `no-cocoa-notification.patch` ) -- https://raw.githubusercontent.com/Homebrew/patches/59e4d32/glib/gnotification-mountain.patch